### ISSUE TITLE
Replace deprecated babel-preset-react-hmre

### ISSUE THREAD

**Issue by wolendranh** [jhen0409/react-chrome-extension-boilerplate#60]
Currently it looks like [babel-preset-react-hmre ](https://github.com/danmartinez101/babel-preset-react-hmre) is being deprecated. Maybe it make sense to investigate and replace it?

**Comment by jhen0409** [comment]
Yeah we may migrate to [RHL v3](https://github.com/gaearon/react-hot-loader/tree/next), it also related to #39. I haven't yet to use it because [some bugs](https://github.com/gaearon/react-hot-loader/issues) blocked me on other projects, so I still not start to do it here, but PR here is welcome.
